gtest_parallel.py

```diff
@@ -382,25 +385,26 @@ def write_to_file(self, save_file):
       pass  # ignore errors---saving the times isn't that important
 
 
-def find_tests(binaries, additional_args):
-  test_count = 0
+def find_tests(binaries, run_disabled_tests, only_failed_and_new, gtest_filter,
+               gtest_color, shard_index, shard_count, repeat, output_dir,
+               additional_args, times, run_command_fn):
```

Comment:
This argument list is... impressive. Maybe consider passing an options object instead? (Because of the duck typing, callers can easily make up a suitable object that has nothing to do with argparse.)

As an added bonus, this would make the diff smaller.